Runtime/Triangulator.cs

```diff
@@ -173,187 +238,207 @@ public class InputData
+                var triangles = output.Triangles;
+                using var circles = new NativeList<Circle>(localPositions.Length, Allocator.Temp);
+                new DelaunayTriangulationJob(output.Status, localPositions.AsArray(), triangles, halfedges).Execute();
+                MarkerDelaunayTriangulation.End();
+
+                NativeArray<Edge> internalConstraints = default;
+                if (input.ConstraintEdges.IsCreated) {
+                    internalConstraints = new NativeArray<Edge>(input.ConstraintEdges.Length/2, Allocator.Temp);
+                    MarkerConstrainEdges.Begin();
+                    new ConstrainEdgesJob {
+                        status = output.Status,
+                        positions = localPositions.AsArray(),
+                        triangles = triangles.AsArray(),
+                        inputConstraintEdges = input.ConstraintEdges,
+                        internalConstraints = internalConstraints,
+                        halfedges = halfedges.AsArray(),
+                        maxIters = sloanMaxIters,
+                    }.Execute();
+                    MarkerConstrainEdges.End();
+
+                    if (localHoles.IsCreated || restoreBoundary) {
+                        MarkerPlantSeeds.Begin();
+                        var seedPlanter = new SeedPlanter(output.Status, triangles, localPositions, circles, internalConstraints, halfedges);
+                        if (localHoles.IsCreated) seedPlanter.PlantHoleSeeds(localHoles);
+                        if (restoreBoundary) seedPlanter.PlantBoundarySeeds();
+                        seedPlanter.Finish();
+                        MarkerPlantSeeds.End();
+                    }
+                }
+
+                if (refineMesh) {
+                    MarkerRefineMesh.Begin();
+                    new RefineMeshJob() {
+                        restoreBoundary = restoreBoundary,
+                        maximumArea2 = 2 * refinementThresholdArea * localTransformation.areaScalingFactor,
+                        minimumAngle = refinementThresholdAngle,
+                        D = concentricShellsParameter,
+                        status = output.Status,
+                        triangles = triangles,
+                        outputPositions = localPositions,
+                        circles = circles,
+                        halfedges = halfedges,
+                        constraints = internalConstraints,
+                    }.Execute();
+                    MarkerRefineMesh.End();
+                }
+
+                MarkerInverseTransformation.Begin();
+                // If an output position list was provided, we need to transform the local positions back to world space.
+                // If none was provided, we can skip this step, as the user doesn't need it.
+                if (output.Positions.IsCreated && preprocessor != Preprocessor.None) {
```

Comment:
We can also remove this since we haven't implemented this feature yet (I mean `output.Positions.IsCreated`).

Reply:
done